entry/src/main/resources/zh/element/string.json

```diff
@@ -0,0 +1,12 @@
+{
+  "string": [
+    {
+      "name": "entry_MainAbility",
+      "value": "entry_MainAbility"
```

Comment:
Please add meaningful string values. If the strings are not required, they can be deleted as well.

Reply:
Done